ui/tests/use-backup-codes.spec.ts

```diff
@@ -23,13 +23,14 @@ test("backup recovery code setup and usage", async ({ context, page }) => {
 
   await expect(page.getByText("Account setup complete")).toBeVisible();
   await expect(page).toHaveScreenshot({ fullPage: true, maxDiffPixels: 500 });
+  // (nsklikas): Is this the expected behavior? The user came from an app, but ended up on a dead page
+  // await finishAuthFlow(page);
```

Comment:
Yes, I think this should work, is this a bug maybe?

Reply:
Ok I will open an issue about it.

Reply:
On a second look, we setup the totp and then we manually navigate to the `setup backup codes` page. So there is no application to redirect the user back to. I will remove the comment.